crates/cairo-coverage/src/input/statement_category_filter.rs

```diff
@@ -10,6 +10,35 @@ use std::iter::once;
 use std::sync::LazyLock;
 
 pub static VIRTUAL_FILE_REGEX: LazyLock<Regex> = LazyLock::new(|| Regex::new(r"\[.*?]").unwrap());
+
+/// This is not the best way to do this, and I'm not proud of it.
+/// However, it is definitely the easiest way to achieve this.
+/// Some functions like `store_temp` are used in many places.
+/// Removing it would eliminate a lot of true positives.
+/// However, users would likely be more frustrated by false negatives.
+static NOT_RELIABLE_LIBFUNCS: LazyLock<HashSet<SimpleLibfuncName>> = LazyLock::new(|| {
+    [
+        "drop",
+        "enable_ap_tracking",
+        "disable_ap_tracking",
+        "struct_deconstruct",
+        "dup",
+        "enum_init",
+        "struct_construct",
+        "store_temp",
+        "return",
```

Comment:
One Sierra return should map to one CASM return, no? How is that libfunc not reliable?

Reply:
Correct me if I'm wrong, but it's because it has no cost. The libfunc that has no cost seems sometimes not to be executed. It need more investigation definitely in https://github.com/software-mansion/cairo-coverage/issues/98

Reply:
It has a cost at CASM level https://docs.cairo-lang.org/how_cairo_works/functions.html#under-the-hood